### DISCUSSION TITLE
这是怎么回事

### DISCUSSION THREAD

**Discussion by ashengjunofficial** [gkd-kit/gkd#645] [Q&A] [answered]
我用的是小米的Civi 1S，澎湃系统1.0.2，重装GKD之后就出现了这样的情况，这可怎么办？有哪位热心的大佬可以帮助一下吗？![Screenshot_2024-06-30-15-49-38-832_li.songe.gkd.jpg](https://github.com/gkd-kit/gkd/assets/174239730/95b1f558-f816-4f41-91ee-54d5f884bc16)

![Screenshot_2024-06-30-15-49-41-797_li.songe.gkd.jpg](https://github.com/gkd-kit/gkd/assets/174239730/3afc468f-6fad-43fd-a246-2a0a85ecb515)

**Comment by lisonge** [comment]
需要日志，否则什么都不知道

**Reply by ashengjunofficial** [reply]
应用列表权限我已经开了，你让我怎么打开日志？

**Reply by lisonge** [reply]
好奇你是不看文档吗？如果我文档写的不清楚可以直接跟我说改或者提 pr 就行了

<https://github.com/gkd-kit/gkd/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=%5BBUG%5D+>

![image](https://github.com/gkd-kit/gkd/assets/38517192/6f2d826e-39cf-4e28-8cf7-398966f85c94)

**Comment by adproqwq** [comment]
尝试给读取应用列表权限

**Reply by ashengjunofficial** [reply]
应用列表权限我已经获取了，可还是那个老样子，怎么办？![Screenshot_2024-07-05-11-00-02-102_com.miui.securitycenter.jpg](https://github.com/gkd-kit/gkd/assets/174239730/a8cf754e-76e5-4c37-9cd3-11aaa8539677)

**Comment by lisonge** [comment, marked as answer]
https://github.com/gkd-kit/gkd/issues/655
